dds/DCPS/MessageTracker.cpp

```diff
@@ -89,13 +96,13 @@ void MessageTracker::wait_messages_pending(const char* caller, const MonotonicTi
   }
   bool loop = true;
   ThreadStatusManager& thread_status_manager = TheServiceParticipant->get_thread_status_manager();
-  while (loop && pending_messages()) {
+  while (loop && pending_messages_i()) {
     switch (done_condition_.wait_until(deadline, thread_status_manager)) {
     case CvStatus_Timeout:
       if (DCPS_debug_level && pending_messages()) {
-        ACE_DEBUG((LM_INFO,
-                   ACE_TEXT("(%P|%t) MessageTracker::wait_messages_pending %T ")
-                   ACE_TEXT("(Redmine Issue# 1446) (caller: %C)\n"),
+        ACE_DEBUG((LM_DEBUG,
+                   ACE_TEXT("(%P|%t) MessageTracker::wait_messages_pending: %T ")
+                   ACE_TEXT("(caller: %C)\n"),
                    ACE_TEXT("Timed out waiting for messages to be transported"),
```

Comment:
I'm not sure this message was correct before your change.  There's a comma before "Timed out" so that's the string inserted as `caller: %C`?  Or am I missing something?